pyqtgraph/exporters/CSVExporter.py

```diff
@@ -79,7 +106,22 @@ def export(self, fileName=None):
                         fd.write(numFormat % d[1][i] + sep)
                     else:
                         fd.write(' %s' % sep)
+
+                    # write data e_y value
+                    if d is not None and len(d)>2 and d[2] is not None and i < len(d[2]):
```

Comment:
I'm not familiar with the exporters here much, can you detail what the data structure of `d` is like, and specifically what you're looking for in this if-statement?